### PR TITLE
Add support for distribution via Swift Package Manager (SwiftPM)

### PR DESCRIPTION
**Pull request by eneko**
This PR adds support for distributing XLForm via Swift Package Manager. This would allow users of Xcode 11 and above to add XLForm as a dependency using Xcode's integrated UI.

Screen Shot 2020-09-17 at 10 58 07 AM

## Summary of Changes
- Add `Package.swift`
- Flatten file structure to ensure consuming applications can find all public header files, without having to manually specify header search paths. Project structure in `XLForm.xcodeproj` has been unaltered.

## Testing
To manually test distribution via XLForm:
1. On a new or existing project in Xcode
1. Go to`File -> Swift Packages -> Add Package Dependency...`
1. Enter url `https://github.com/eneko/XLFom`
1. Choose branch `master`
1. Use XLForm in your app 🎉

### CODE REVIEW

**Comment by eneko** [comment]
@mats-claassen could you please re-run the CI build on Travis? CocoaPods was timing out this morning, but it's working again. Build is passing on my end: https://travis-ci.com/github/eneko/XLForm/builds/185000478

**Comment by mats-claassen** [comment]
Hi @eneko do you mind also adding a note to the Readme on how to install using SPM?

**Comment by eneko** [comment]
Hi @mats-claassen, sounds good. PR updated.

File: README.md [hunk 1]
```diff
@@ -1125,6 +1125,28 @@ You can make this happen using the `textFieldMaxNumberOfCharacters` and the `tex
...
 
+## Swift Package Manager
+
+Starting with Xcode 11, Swift Package Manager is the recommended and preferred way for installing dependencies in Xcode projects. Installing dependencies via SwiftPM does not require the application nor dependencies to be written in Swift.
+
+To add XLForm to your project using SwiftPM follow these steps:
+
+1. Open your project in Xcode
+1. In the main menu, select File -> Swift Packages -> Add Package Dependency...
+1. In the window, enter the package url https://github.com/xmartlabs/XLFom
```
Comment (mats-claassen) [inline README.md:1136]:
```suggestion
1. In the window, enter the package url https://github.com/xmartlabs/XLForm
```

**Comment by eneko** [comment]
Thank you, @mats-claassen 🎉

When you have a chance, could you tag a new release? Both 4.1.1 or 4.2.0 would work.

**Pull request merged**